lambdas/thumbnail/index.py

```diff
@@ -47,7 +54,146 @@
     'additionalProperties': False
 }
 
-SUPPORTED_BROWSER_FORMATS = {'JPEG', 'GIF', 'PNG'}
```

Comment:
Need to bring this back: we want to keep the original image format, but only as long as it's well-supported.

(The reasoning is, converting PNG to JPEG can add artifacts; converting JPEG to PNG can make it unnecessarily large, etc.)